### PR TITLE
some updates/ fixes/ wicket 8.9

### PR DESCRIPTION
**Pull request by kbachl**

### CODE REVIEW

**Comment by kbachl** [comment]
@martin-g 
have a look and in case you are ok resolve the pom.xml with

```
<artifactId>xstream</artifactId>
<version>1.4.13</version>
```

File: pom.xml [hunk 1]
```diff
@@ -212,16 +207,23 @@
 				<version>${jackrabbit.version}</version>
 			</dependency>
 			<!--+ | HELPERS + -->
+			<!-- Attention: do not user a *-java7 version of xstream! - it is only for pure java 7 runtimes, see https://github.com/x-stream/xstream/issues/112		-->
```
Comment (martin-g) [inline pom.xml:210]: s/user/use/

File: pom.xml [hunk 2]
```diff
@@ -212,16 +207,23 @@
...
 				<groupId>com.thoughtworks.xstream</groupId>
 				<artifactId>xstream</artifactId>
-				<version>1.2.2</version>
+				<version>1.4.13</version>
 			</dependency>
+<!--			<dependency>-->
+<!--				<groupId>xpp3</groupId>-->
+<!--				<artifactId>xpp3_min</artifactId>-->
+<!--				<version>1.1.4c</version>-->
+<!--			</dependency>-->
```
Comment (martin-g) [inline pom.xml:220]: If `org.ogce:xpp3` works then remove `xpp3:xpp3_min` dependency completely

File: pom.xml [hunk 3]
```diff
@@ -391,6 +372,27 @@
...
+				<executions>
+					<execution>
+						<id>enforce-maven</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<rules>
+								<requireMavenVersion>
+									<version>3.6.3</version>
```
Comment (martin-g) [inline pom.xml:388]: not really required, but it is OK

**Review by martin-g — APPROVED** [review]

**Pull request merged**